src/lsptoolshost/roslynLanguageServer.ts

```diff
@@ -412,6 +413,10 @@ export class RoslynLanguageServer {
             args.push('--logLevel', logLevel);
         }
 
+        if (this.additionalExtensionPaths.length > 0) {
+            args.push('--additionalExtensions', `"${this.additionalExtensionPaths.join(',')}"`);
```

Comment:
I think this is fine to merge, even if the server doesn't have this property, right?  Since it'll only activate if there's another extension that provides this path.

Reply:
That's what I expect. I just want to make sure that this is the flag and package.json names we want.